sway-lib-std/src/ecr.sw

```diff
@@ -51,6 +51,7 @@ pub enum EcRecoverError {
 ///     assert(public_key.bits()[1] == pub_lo);
 /// }
 /// ```
+#[deprecated(note = "Please use new crypto module")]
```

Comment:
```suggestion
#[deprecated(note = "std::ecr has been replaced by std::crypto, and is no longer maintained")]
```

Ideally we would point to the specific function that replaces `ec_recover`, i.e,, <name_of_signature_type>::recover, but since we have support for 3 different signature types (maybe with more to come?) I don't see a simple way to do that.